app/models/application_record.rb

```diff
@@ -21,11 +21,12 @@ def log_errors
   # Condition for rollout of mass addition of validation rules.
   # Cached for performance.
   def mass_validation_enabled?
-    if AppConfig.table_exists? # for migrations previous to AppConfig creation
-      @@mass_validation_enabled ||= AppConfigHelper.get_app_config(AppConfig::ENABLE_MASS_VALIDATION) || false # rubocop:disable Style/ClassVars
+    # for migrations previous to AppConfig creation
+    if AppConfig.table_exists?
+      @@mass_validation_enabled = !!AppConfigHelper.get_app_config(AppConfig::ENABLE_MASS_VALIDATION) if @@mass_validation_enabled.nil? # rubocop:disable Style/ClassVars,Style/DoubleNegation
```

Comment:
Actually you can get rid of double negation and go back to `|| false`...

Reply:
agree, but I'd prefer not to re-test

Reply:
just hate disabling alert inline

Reply:
in general, do you think `!!` is a bad pattern? maybe we should review all the "skips" we have in the codebase in the next eng lunch.

Reply:
No, i like it ...